sdk/eventhub/eventhubs-checkpointstore-blob/tsconfig.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "declarationDir": "./typings",
     "outDir": "./dist-esm",
-    "downlevelIteration": true
+    "downlevelIteration": true,
+    "lib": ["ES2018.AsyncIterable"]
```

Comment:
I believe our guidelines encourage us to not use the `lib` field in tsconfig.json. What is the error message you're getting when you see this?

Reply:
I don't remember the error message that I was getting - the ci pipelines that ran in the past have gone now. I am running the pipeline again after removing this config, to see if it still gives error

Reply:
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=722578&view=logs&j=1c9290f0-7664-5780-0e82-5fab6e3e2db5&t=d499c3a2-fe07-5c11-f4a0-44595fb56903

Reply:
You should be able to add a reference comment to whichever file imports `@azure/core-paging` that looks like this:
`/// <reference lib="esnext.asynciterable" />`.

I'm curious though, do you know why this wasn't giving us problems before?

Reply:
Hmmm, min-max makes the tests folder a new npm project, probably that's causing it?

Reply:
> Hmmm, min-max makes the tests folder a new npm project, probably that's causing it?

Ah, I didn't realize it was treated like a new project. That makes sense. Then I think we should add the reference comment. keyvault-keys does this as well, it makes it so our users don't have to add it to their tsconfig.json.

Reply:
We need to do that to service-bus too then.

Reply:
The core-paging dependency comes from storage-blob. As discussed with Chris, opened an issue to not take dependency on storage-blob. https://github.com/Azure/azure-sdk-for-js/issues/13667

Reply:
Also opened this issue on storage blob - #13669

Reply:
Hey @KarishmaGhiya, I believe service-bus was also updated with the same in the other min-max PR, can you take care of that too?

Reply:
If we add the reference comment, then we will need to make sure that the asynciterator polyfill is loaded into this package. Adding a lib reference to `es2018.asynciterator` will mean that anyone who uses this package will end up with asynciterator definitions as well, but that's appropriate if we're polyfilling it.